### ISSUE TITLE
Does this work anymore :( ?

### ISSUE THREAD

**Issue by DukeOfDarkness** [Lordmau5/node-spotify-downloader#60]
The Tool downloads only a cover and keep stuck at downloading the first song for me ... =(
(NodeJS 4.4.4)

**Comment by jonaaa20** [comment]
Let me explain, spotify is "blocking" the tool, sending some wrong urls, so, sometimes it works, sometimes not...

**Comment by kimgkimg** [comment]
Does not appear to be working now.  Downloads just hang on the first track.

**Comment by daniDLL** [comment]
Same here, it doesnt work anymore :'(

**Comment by jonaaa20** [comment]
Like i've said, sometimes it works, sometimes not...

**Comment by skypce** [comment]
can you encrypt your solution, this way spotify did not apply a fix

**Comment by psycholyzern** [comment]
I have tried over 10 playlist, twice for each, and none of them were downloaded  So I guess this project is totally not working now.

**Comment by Pyuki** [comment]
same here

**Comment by jonaaa20** [comment]
indeed, the tool is not working (we are still looking for a fix), for now, use another way to download your musics like deezloader or musicforyou.ga or youtube until we fix the problem...

**Comment by rdtraxx** [comment]
We hope you find the fix asap... THANKS

**Comment by XDjackieXD** [comment]
(slight hint for ones with a bit of rust knowledge: librespot. I did a thing with it but sharing such code would probably kill all open-source libspotify clones :/)

**Comment by dj-thd** [comment]
It seems the error comes from the used library, that is "node-spotify-web". It stucks at the call to function "play()"

**Comment by NoLooseEnds** [comment]
Same problem here :/

**Comment by Dan-Wood** [comment]
They've started using Google ReCaptcha for logins now :)

Error logging in... (Error: fallback: Invalid captcha value supplied)

**Comment by hmborges** [comment]
The same here, "Error logging in... (Error: fallback: Invalid captcha value supplied)" :(

**Comment by jonaaa20** [comment]
I'll say again:

1- Spotify blocked the tool (so it will NOT WORK)
2- Spotify is using reCaptcha on logins (so it will NOT WORK TOO!)

So... use any other alternative like mp3fy, musicforyou, youtube, soundcloud, deezloader or if you have a premium account on spotify, try the spotify-ripper.

**Comment by Lordmau5** [comment]
I am actually investigating https://github.com/FrontierPsychiatrist/node-spotify over the past few weeks, but since I had university applications to do, I couldn't really look into it much.

If we get the code moved over to that, then a premium account **is required.**
On the other hand, it would open up the doors for 320kbit audio.

I should be done with the applications this weekend which means I can look into it and see if it's possible to move the code over :)

Sorry to keep all you guys waiting.

**Comment by mR12KL** [comment]
Lordmau5 - any update? We're all eager! :)

**Comment by rdtraxx** [comment]
Somebody knows if there is any other solution?

**Comment by anthuan44** [comment]
@rdtraxx Spotify-ripper is your only hope apart from MP3FY, but you need a premium account in order to use it.

**Comment by mR12KL** [comment]
https://gitter.im/Lordmau5/node-spotify-downloader - track the status of the ripper here! We have our best minds (MP3FY and Lordmau5) working on it!

**Comment by kevienfox** [comment]
;((((

**Comment by Grinninga** [comment]
I hope this thing works again at some time.. :)
Tell us when it's time!

**Comment by ohemb** [comment]
Regarding similar services - anyone with knowledge about what happened to MP3FY? Cheers

**Comment by Grinninga** [comment]
@ohemb it's the same with our downloader.. spotify themselves made a captcha which u have to pass when u login, and i think it's the same error at mp3fy. but i don't know exactly

**Comment by mR12KL** [comment]
@Grinninga @ohemb you can ask MP3FY himself - gitter.im/Lordmau5/node-spotify-downloader
